app/javascript/components/widgets/forest-change/fires-alerts-cumulative/selectors.js

```diff
@@ -177,12 +184,12 @@ export const getLegend = createSelector(
         label: `${moment(end.date).format('YYYY')}`,
         color: colors.main
       },
-      ...(compareYear && {
-        compare: {
-          label: `${compareYear}`,
-          color: '#49b5e3'
-        }
-      }),
+      // ...(compareYear && {
```

Comment:
comments?

Reply:
Yeah - it needs refactoring to iterate through the array of compare years so that it works whether a single year, or all years are selected.

Reply:
Removed